BUILD/familiars/drop.dat

```diff
@@ -2,6 +2,8 @@
 #
 # First we grab up to small amount of various specific drops.
 #
+# every 2 cookbookbat ingredients generate a size 1 density 7 food. prioritize having enough to craft 1.
+Cookbookbat	item:Vegetable of Jarlsberg<2;item:Yeast of Boris<2;item:St. Sneaky Pete's Whey<2
```

Comment:
Believe this acts as an AND. Is the intent to have 2 of all ingredients or simply 2 of any ingredient?

Reply:
Yep, all on one line acts as an AND. I found it easier to think about by inverting the logic in my head. This will skip the Cookbookbat if you have 2 of any ingredient (De Morgan's law means AND inverts to OR). Then lower where I have multiple lines is OR, so it'll want to generate 4 of everything before it'll stop using Cookbookbat.

Think it's worth adding another explanatory comment?

Reply:
Nah it makes total sense when I take a second to think about it